### PR TITLE
Regionalgam tools

### PR DESCRIPTION
**Pull request by Alanamosse**
This is a new attempt to add regionalgam tools.
Rebased with master.

Tests are modified to compare raw values instead of binary plot files.

### CODE REVIEW

**Comment by bgruening** [comment]
@Alanamosse traveling right now. Can you ping me here and everywhere if you need to review / if this is green?

**Comment by Alanamosse** [comment]
@bgruening Ok 
Thank you and good journey !

**Comment by Alanamosse** [comment]
@bgruening Green at last ! :)

File: tools/regionalgam/ab-index.R [hunk 1]
```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env Rscript
+#library("getopt")
```
Comment (bgruening) [inline tools/regionalgam/ab-index.R]: remove

Comment (Alanamosse) [reply tools/regionalgam/ab-index.R]: Good catch :+1:

**Pull request merged**